trulens_eval/trulens_eval/pages/Evaluations.py

```diff
@@ -136,6 +134,13 @@ def render_record_metrics(app_df: pd.DataFrame, selected_rows: pd.DataFrame):
         delta_color="inverse",
     )
 
+        # Define a function to extract record metadata from each row
+def extract_metadata(row):
+    try:
+        record_json = json.loads(row['record_json'])
```

Comment:
What was the decoding error that was happening here?

Reply:
Think this was unneccessary. Removed the try/except